packages/elements-core/src/components/TryIt/Auth/Auth.tsx

```diff
@@ -41,7 +41,7 @@ export const TryItAuth: React.FC<TryItAuthProps> = ({ operationSecurityScheme: o
         title: 'Security Schemes',
         children: filteredSecurityItems.map(auth => ({
           id: `security-scheme-${auth.key}`,
-          title: getReadableSecurityName(auth),
+          title: getReadableSecurityName(auth, shouldIncludeKey(filteredSecurityItems, auth.type)),
```

Comment:
What do you think about changing the menu display name to include keys as well instead of only `API Key`? (I guess that would be line 26 in this file
![Screen Shot 2021-08-10 at 12 59 53 PM](https://user-images.githubusercontent.com/58433203/128855519-c3b00891-c5d8-43ef-a1c9-616d264c198c.png)

Reply:
I intentionally didn't add it there as I thought it might make it messy and that it may be batter to kepp button with generic name. But we could do that if that's something worth to do.